pkg/lib/config/secret.go

```diff
@@ -265,8 +265,8 @@ const (
 	// nolint: gosec
 	OAuthClientCredentialsKey SecretKey = "oauth.client_secrets"
 	// nolint: gosec
-	CaptchaCloudflareCredentialsKey  SecretKey = "captcha.cloudflare"
-	WhatsappOnPremisesCredentialsKey SecretKey = "whatsapp.on-premises"
+	LegacyCaptchaCloudflareCredentialsKey SecretKey = "captcha.cloudflare"
```

Comment:
Rename LegacyCaptchaCloudflareCredentialsKey to Deprecated_CaptchaCloudflareCredentialsKey